### ISSUE TITLE
[Bug]when talkback is enabled, it says loading repeatedly.

### ISSUE THREAD

**Issue by paxcoder** [mozilla-mobile/fenix#17080]
Enable talkback, and go to any website.

### Device information

* Android device: ?
* Fenix version: ?

**Comment by DiegoARL38** [comment]
This happens doing multiple, different things.
* Scrolling a website.
* Writing a comment on GitHub, after each character.
* Opening the context menu on web content.
* Changing application and coming back, either via Task Switcher or shortcut.
* zooming a webpage.

**Comment by paxcoder** [comment]
> This happens doing multiple, different things.
> * Scrolling a website.
> * Writing a comment on GitHub, after each character.
> * Opening the context menu on web content.
> * Changing application and coming back, either via Task Switcher or shortcut.
> * zooming a webpage.
> 
> Using a Xiaomi Redmi Note 8 with MiUI 11.0.12 (Android 9).
> Nightly 201215 17:01 (Build #2015781579)
> AC: 70.0.20201214143056, 5cea9c9a6
> GV: 85.0a1-20201213093323
> AS: 67.2.0

I have Xiaomi redmi note 9

**Comment by paxcoder** [comment]
And what?

**Comment by DiegoARL38** [comment]
> And what?

About what?

**Comment by paxcoder** [comment]
This problem

**Comment by DiegoARL38** [comment]
They'd need to prioritise the work, and we're on holiday season, so I wouldn't really expect any news until January.

**Comment by laiam** [comment]
going to bump this because it's the number one major issue preventing me from upgrading, firefox is literally the only browser I have left that worked with talkback the way I needed it to, i've had to downgrade back to firefox 68 and my phone keeps yelling at me that it's a security risk as an outdated app.

aside from this all my other complaints about the "new and supposedly improved" firefox are stylistic in nature and f*** all if I can do anything to convince anyone to do anything about that.

**Comment by mhoye** [comment]
While this issue has been queued up for next steps by the team, this is a polite reminder that this is our professional working environment as much as it is an issue tracker, and that Mozilla's community participation guidelines apply. Thank you all.

**Comment by gabrielluong** [comment]
Clearing assignee since I don't believe this is being worked on.

**Comment by mcarare** [comment]
I cannot reproduce this issue. Tried on Huawei, Nokia, Xiaomi with different Android versions. Can QA also try to reproduce this? Thank you!

**Comment by DiegoARL38** [comment]
I'm not from QA, but the bug is still present, trying what I outlined in a prior comment.

Using a Xiaomi Redmi Note 8 with MIUI 12.0.4, Android 10.
Nightly 210425 17:01 (Build #2015806731)
AC: 90.0.20210421164305, b8f35b778
GV: 90.0a1-20210420095122
AS: 74.0.1

**Comment by mcarare** [comment]
@DiegoARL38 I am not denying it is reproducible on the device you specified, the issue is that I cannot try to fix it unless I can reproduce it. Can you also check what Talkback version you have? Thank you!

**Comment by DiegoARL38** [comment]
@mcarare I'm using TalkBack 9.1.

**Comment by mcarare** [comment]
I tested on the same version and I cannot reproduce it.
Also, this message is not something we explicitly use in the app. Tagging @eeejay, maybe he has more info if this announcement is something generated by GeckoView.

**Comment by ndarilek** [comment]
I'm able to duplicate this on a Pixel 4A 5G with all updates applied, 
Nightly as of yesterday, and the absolute latest TalkBack. I can find 
the exact version if needed. Here are my steps:


1. Visit https://www.kxan.com/weather/forecast/todays-forecast/

2. Find the title h1.

3. Use TalkBack's "Read from next item" command. I think the quickest 
way to do this is to swipe up/right, then tap "Read from next item." I 
have a custom shortcut defined for this and don't recall the default.


Firefox reads a bit, then says "Loading" and stops.


I don't think this is tied to highly specific component versions. I 
stopped using Firefox a few months back because of this bug. It doesn't 
happen with Brave, so it isn't a) a recent regression and b) something 
TalkBack does universally.


Thanks. Please let me know if I can provide further details.

**Comment by mcarare** [comment]
@ndarilek Thank you for your report. I will move the details you provided to a new issue because what you described is slightly a different issue than this. This mentions the "loading" message is repeated multiple times, while the situation you described is due to the fact that the webpage is not scrolled when using the "Read from next item" command. 
Can you tell me if you've encountered the issue described in this topic?

**Comment by ndarilek** [comment]
Maybe it's the same issue, maybe not. Maybe visually it manifests as not scrolling, but for me, TalkBack says "Loading" again and again. I don't remember specifically at what point "Loading" is spoken but it definitely is.

**Comment by DiegoARL38** [comment]
In my case, it does scroll. However, it only happens once, before being interrupted by the loading message.

**Comment by AndiAJ** [comment]
Hi, managed to **reproduce** after opening `theverge.com` and `cnn.com` on the latest **`Nightly 90.0a1 5/4`** using the following devices:
• Google Pixel 3a (Android 11)
• OnePlus A3 (Android 6.0.1)

► [Video](https://user-images.githubusercontent.com/51314259/116975180-e4add900-acc7-11eb-8d31-4b0c8e261c80.mp4)

**Comment by rocketsroger** [comment]
Looking into this issue I found that the "loading" announcement was meant to announce to accessibility users that the page is loading.  However, looks like when there are small loads it also announce loading causing constant audible "loading".

The fix will need to be in Android components side.  In the worst case I'll remove the "loading" announcement completely but if possible I'll try to limit the announcement to only page loads.

**Comment by rocketsroger** [comment]
Confirmed fix is in Nightly.  Please verify.  Thanks!

**Comment by sflorean** [comment]
Verified as fixed on Nightly 5/28 with Samsung Note 10 (Android 11).
